packages/vats/src/core/basic-behaviors.js

```diff
@@ -304,15 +302,15 @@ export const produceStartGovernedUpgradable = async ({
         governedParams,
         timer: chainTimerService,
         contractGovernor,
-        economicCommitteeCreatorFacet,
+        committeeCreatorFacet,
       },
     );
     const kit = harden({ ...facets, label });
     contractKits.init(facets.instance, kit);
 
     await E(diagnostics).savePrivateArgs(kit.instance, privateArgs);
     await E(diagnostics).savePrivateArgs(kit.governor, {
-      economicCommitteeCreatorFacet: await economicCommitteeCreatorFacet,
+      committeeCreatorFacet: await committeeCreatorFacet,
```

Comment:
I don't understand how we can change this without changing the `start` function of the contract governor.

Reply:
This just writes the value for "disaster recovery". It's never read. 

https://github.com/Agoric/agoric-sdk/blob/d8fc335f0988d454b99adcf259a046c207d683ad/packages/vats/src/core/basic-behaviors.js#L120-L126

It's not really a privateArgs and I think it needn't be saved because the committeeCreatorFacet is not precious, but I wasn't confident enough in that to make the change.